src/bors/mod.rs

```diff
@@ -105,3 +105,17 @@ pub struct RepositoryState<Client: RepositoryClient> {
     pub permissions: ArcSwap<UserPermissions>,
     pub config: ArcSwap<RepositoryConfig>,
 }
+
+/// A comment that can be posted to a pull request.
+pub struct Comment {
+    text: String,
```

Comment:
I could't find a use for the metadata, as currently homu just serialize the entire comment to the html tag. Once there is a concrete usecase I will update this